docs/learn/feature-definitions.md

```diff
@@ -36,11 +48,17 @@ class VideoFeatureSpec(BaseFeatureSpec[VideoIds]):
 
 `BaseFeatureSpec` is a [Pydantic](https://docs.pydantic.dev/latest/) model, so all normal Pydantic features apply.
 
+Feature specs now support an optional `metadata` dictionary for attaching ownership, documentation, or tooling context to a feature.
+This metadata **never** influences graph topology or version hashes, must be JSON-serializable, and is immutable once the spec is created.
```

Comment:
This metadata does not influence graph topology or the versioning system. 

---

That's enough, the rest redundant. The entire feature spec is frozen btw, just like all other models (unless something has slipped).